caesar_cipher/run-cli.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'lib/cipher'
+
+user_message = ARGV[0]
+user_input_number = ARGV[1].to_i
+
+print "Encrypted text is : "
+puts Cipher.new(user_input_number).encrypt(user_message)
```

Comment:
@zzokki81 Currently this doesn't work very well:

```
$ ruby caesar_cipher/run-cli.rb                                                 
Encrypted text is : Traceback (most recent call last):
	1: from caesar_cipher/run-cli.rb:7:in `<main>'
caesar_cipher/lib/cipher.rb:10:in `encrypt': undefined method `each_char' for nil:NilClass (NoMethodError)

$ ruby caesar_cipher/run-cli.rb "Some text here" 42                             
Encrypted text is : c}u ?u?? xu?u

$ ruby caesar_cipher/run-cli.rb "Some text here" 1000                           
Encrypted text is : Traceback (most recent call last):
	4: from caesar_cipher/run-cli.rb:7:in `<main>'
	3: from caesar_cipher/lib/cipher.rb:10:in `encrypt'
	2: from caesar_cipher/lib/cipher.rb:10:in `each_char'
	1: from caesar_cipher/lib/cipher.rb:21:in `block in encrypt'
caesar_cipher/lib/cipher.rb:21:in `chr': 1057 out of char range (RangeError)
```

Reply:
@zzokki81 all these 3 examples still throw an error. The caller needs to handle any possible exceptions.